client/ayon_hiero/plugins/create/create_shot_clip.py

```diff
@@ -192,23 +227,44 @@ def get_instance_attr_defs(self):
             )
         ]
         if self.product_type == "plate":
-            instance_attributes.extend([
-                BoolDef(
-                    "vSyncOn",
-                    label="Enable Vertical Sync",
-                    tooltip="Switch on if you want clips above "
-                            "each other to share its attributes",
-                    default=True,
-                ),
-                EnumDef(
-                    "vSyncTrack",
-                    label="Hero Track",
-                    tooltip="Select driving track name which should "
-                            "be mastering all others",
-                    items=gui_tracks or ["<nothing to select>"],
-                ),
-            ])
-
+            # Review track visibility
+            current_review = instance.creator_attributes.get("review")
+            if current_review is None:
+                current_review = False
+
+            review_source = instance.creator_attributes.get("reviewableSource")
+            if review_source:
+                instance.creator_attributes["review"] = True
+                current_review = True
+
+            instance_attributes.extend(
+                [
+                    BoolDef(
+                        "review",
+                        label="Review",
+                        tooltip="Switch to reviewable instance",
+                        default=False,
+                    ),
+                    EnumDef(
+                        "reviewableSource",
+                        label="Reviewable Source",
+                        tooltip=(
+                            "Generate preview videos on fly, if '< none >' "
```

Comment:
This tooltip is outdated.

Reply:
you are right, 7d4323b49d974f450af101039a27882067b6f2b9